sympy/core/tests/test_relational.py

```diff
@@ -718,6 +718,30 @@ def test_issue_10927():
     assert str(Eq(x, -oo)) == 'Eq(x, -oo)'
 
 
+def test_issues_13081_12583_12534():
+    # 13081
+    r = Rational('905502432259640373/288230376151711744')
+    assert (r < pi) is S.false
+    assert (r > pi) is S.true
+    # 12583
+    v = sqrt(2)
+    u = sqrt(v) + 2/sqrt(10 - 8/sqrt(2 - v) + 4*v*(1/sqrt(2 - v) - 1))
+    assert (u >= 0) is S.true
+    # 12534
+    G = [Rational(pi.n(i)) > pi for i in (100, 101)]
```

Comment:
Maybe some comments here explaining the digits of pi near 100 and how the rounding works there could be helpful.